pkgs/development/libraries/nss/default.nix

```diff
@@ -14,7 +14,7 @@ in stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "mirror://mozilla/security/nss/releases/NSS_${underscoreVersion}_RTM/src/${pname}-${version}.tar.gz";
-    sha256 = "0q8m9jf6zgkbhx71myjb7y0gcl5ib3gj6qkl9yvdqpd6vl6fn2ha";
+    sha256 = "sha256-vMgawzrrTsrRgtwh400sl7zBSP17nnb4W+u4kkBakng=";
```

Comment:
Please, don't use SRI hashes in official nixpkgs yet.  See #89275.

Reply:
Fixed.